### PR TITLE
[2.0] Update example configuration files to fix various problems.

### PR DESCRIPTION
**Pull request by SaberUK**
This is not finished yet &mdash; this pull request is to allow people to comment on the changes I have made.

See the commit message for a list of all changes.

### CODE REVIEW

**Comment by SaberUK** [comment]
I also think we should remove `docs/aliases/ircservices.conf.example` and `docs/aliases/neostats.conf.example` as both of those services packages are dead. What do you think, @attilamolnar?

**Comment by attilamolnar** [comment]
@SaberUK I agree, I've purged them.

File: docs/conf/modules.conf.example [hunk 1]
```diff
@@ -1589,25 +1607,25 @@
 
 #-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#
 # Servprotect module: Provides support for Austhex style +k /
-# UnrealIRCD +S services mode.
+# UnrealIRCd +S services mode.
```
Comment (attilamolnar) [inline docs/conf/modules.conf.example]: We should explain what the mode does here, pointing to another implementation is not very helpful

File: docs/conf/motd.txt.example [hunk 2]
```diff
@@ -13,7 +13,7 @@
        //\
        V  \    WELCOME TO AN INSPIRCD NETWORK
         \  \_    If you see this, I am probably new.
-         \,'.`-.   If I'm not new, my owner is lazy.
+         \,'.`-.   If i'm not new, my owner is lazy.
```
Comment (attilamolnar) [inline docs/conf/motd.txt.example]: Oversight here and same thing in opermotd.txt.example

File: docs/conf/helpop.conf.example [hunk 3]
```diff
@@ -78,36 +78,36 @@ LOCKSERV       UNLOCKSERV">
...
  R            Blocks private messages from unregistered users
-              (requires services account module).
- S            Strips mIRC color/bold/underline codes out of private
-              messages to the user (requires stripcolor module).
+              (requires the services account module).
+ S            Strips color/bold/underline codes out of private
+              messages to the user (requires the stripcolor module).
  W            Receives notification when a user uses WHOIS on them
-              (IRCop only, requires showwhois module).">
+              (IRCop only, requires the showwhois module).">
```
Comment (attilamolnar) [inline docs/conf/helpop.conf.example]: "IRCop" was not replaced here, deliberate or mistake?

Comment (SaberUK) [inline docs/conf/helpop.conf.example]: Mistake; fixed.

**Comment by SaberUK** [comment]
Closing this because it would probably cause too many merge conflicts with master.

**Pull request closed without merging**